### PR TITLE
2.80 merge support

### PR DESCRIPTION
**Pull request by bdotq**
updated for 2.8 support of merging objects. Tested exporting OGRE v2 meshes.

### CODE REVIEW

File: io_ogre/ogre/mesh.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ def dot_mesh( ob, path, force_name=None, ignore_shape_animation=False, normals=T
                     _remap_normals_.append( n )
                     _face_indices_.append( F.index )
 
-                    x,y,z = swap(ob.matrix_world * v.co)        # xz-y is correct!
```
Comment (paroj) [inline io_ogre/ogre/mesh.py]: is this change on purpose?

File: io_ogre/ogre/mesh.py [hunk 2]
```diff
@@ -579,7 +579,7 @@ def get_or_create_modifier(obj, modifier_name):
 
                 for vidx, v in enumerate(_remap_verts_):
                     pv = skey.data[ v.index ]
-                    x,y,z = swap( ob.matrix_world * pv.co - ob.matrix_world * v.co )
```
Comment (paroj) [inline io_ogre/ogre/mesh.py]: is this change on purpose?

File: io_ogre/ogre/scene.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def dot_scene(path, scene_name=None):
     for ob in bpy.context.scene.objects:
         if ob.subcollision:
             continue
-        if not config.get("EXPORT_HIDDEN") and ob.hide:
+        if not config.get("EXPORT_HIDDEN") and ob.hide_viewport:
```
Comment (paroj) [inline io_ogre/ogre/scene.py]: is this change on purpose?

Comment (bdotq) [reply io_ogre/ogre/scene.py]: yes

File: io_ogre/ogre/mesh.py (hunk 2 above)
Comment (bdotq) [reply io_ogre/ogre/mesh.py]: hmmmmmmmmmmm .. I don't remember making this change ...

File: io_ogre/ogre/mesh.py (hunk 1 above)
Comment (bdotq) [reply io_ogre/ogre/mesh.py]: hmmmmmmmmmmm .. I don't remember making this change ...

**Comment by paroj** [comment]
merged manually d79525c220ed55fb4ae69b411e1fd324143eb803
